.vscode/launch.json

```diff
@@ -569,6 +571,22 @@
                     "ignoreFailures": true
                 }
             ]
+        },
+
+        {
+            "name": "Camera Controller (Linux)",
```

Comment:
Instead of createing dedicated entry for every app (what about `out/linux-x64-camera-controller-no-thread` or other variations?) there is a "Run application (Linux x64)" or "Run application (Darwin x64)" entry. As it is right now it will not find `camera-controller`, but you can change app name `chip-camera-controller` or try improve app filtering in the `outAppLinux` input command.

Reply:
changes app name to chip-camera-controller